src/main/java/io/strimzi/kafka/bridge/mqtt/core/MqttKafkaMapper.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.bridge.mqtt.core;
+
+import io.strimzi.kafka.bridge.mqtt.utils.MappingRule;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Comparator;
+
+/**
+ * Responsible for handling all the topic mapping.
+ *
+ * @see MappingRule
+ */
+public class MqttKafkaMapper {
+
+    private static String DEFAULT_KAFKA_TOPIC = "messages_default";
+    private ArrayList<MappingRule> rules;
+
+    /**
+     * Constructor
+     *
+     * Creates a new instance of MqttKafkaMapper.
+     */
+    public MqttKafkaMapper(){
+        rules = new ArrayList<>();
+        rules.add(new MappingRule("building_{building}_room_{room}", "building/{building}/room/{room}/#", null));
+        rules.add(new MappingRule("sensor_data", "sensors/+/data", null));
+        rules.add(new MappingRule("devices_{device}_data", "devices/{device}/data", null));
+        rules.add(new MappingRule("fleet_{vehicle}", "fleet/{fleet}/vehicle/{vehicle}/#", null));
+        rules.add(new MappingRule("building_{building}_others","building/{building}/#", null));
+        rules.add(new MappingRule("sensor_others", "sensors/#", null));
+        rules.add(new MappingRule("building_others", "building/#", null));
+        rules.sort(Comparator.comparing(MappingRule::getMqttTopicPatternLevels).reversed());
+    }
```

Comment:
Why does the constructor fill all these hard coded rules? They should be loaded from a JSON file as we agreed or anyway if not loading from a file yet, you should be able to fill different rules during different tests to check that the logic works.

Reply:
loading during the tests should be fine. But I am already working on loading configurations from application.proprieties file.

Reply:
for this PR, I am loading the test cases from a json.

Reply:
I am not sure that for testing this we need a JSON file with a lot of rules to be loaded.
I think we should have more different tiny tests focused on some rules that you can pass to the `MqttKafkaMapper` easily. In the end, it's a list of `MappingRule`. So each test should fill a rules list with a few rules and then testing just them.

Reply:
Use `List<MappingRule>`

Reply:
@ppatierno I am now using List everywhere I was using arraylist.